app/src/main/java/com/lastaoutdoor/lasta/data/api/autocomplete/RadarRepositoryImpl.kt

```diff
@@ -0,0 +1,46 @@
+package com.lastaoutdoor.lasta.data.api.autocomplete
+
+import com.lastaoutdoor.lasta.models.api.RadarSuggestion
+import com.lastaoutdoor.lasta.repository.api.RadarRepository
+import com.lastaoutdoor.lasta.utils.Response
+import javax.inject.Inject
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.suspendCancellableCoroutine
+import retrofit2.Call
+import retrofit2.Callback
+
+class RadarRepositoryImpl @Inject constructor(private val radarApiService: RadarApiService) :
+    RadarRepository {
+
+  private val layers: String = "county,locality,state"

```

Comment:
is there a typo or is it supposed to say "county" instead of "country"? Also, could make this a constant!

Reply:
No, it is not a typo, the county is the equivalent of "Cantons" (the API was developed in the US so they just translated that to this I think)